lang/python/python3/Makefile

```diff
@@ -184,7 +187,10 @@ endif # CONFIG_PACKAGE_python3-setuptools
 
 ifdef CONFIG_PACKAGE_python3-pip
 define Build/Compile/python3-pip
-	$(STAGING_DIR_HOSTPKG)/bin/pip3 install \
+	$(HOST_PYTHON3_PIP) \
+		--disable-pip-version-check \
+		--cache-dir "$(DL_DIR)/pip-cache" \
```

Comment:
[curios] does this directory get created if it does not exist ?

Reply:
It should (and it did during my testing).